spec/factories/tasks.rb

```diff
@@ -1,5 +1,8 @@
 FactoryBot.define do
   factory :task do
-    
+		title { "first_title" }
+		content { "first_content" }
+		status { "todo" }
```

Comment:
enumの値はStringではなくSymbolで定義しましょう。
```suggestion
		status { :todo }
```

Reply:
25effde でenumの値をstringからSymbolに修正しました